### PR TITLE
login page refactor style

### PR DESCRIPTION
**Pull request by camiriveron**
connects to #36 

comps: https://www.dropbox.com/work/Design%3ADev%20filedropper/Franklin/Login?preview=franklin-login.jpg

### CODE REVIEW

File: src/scss/type.scss [hunk 1]
```diff
@@ -108,13 +108,31 @@ h6 {
...
-    margin-bottom: 1em;
-    color: #FFF;
-    line-height: 1.75em;
+    font-family: 'Lato', sans-serif;
+    font-size: 0.5em; //12px desk
+    color: $franklin-softgray;
 }
 
-.color-white {color: #FFF!important;}
-.color-green{color: #5d8f3c!important;}
```
Comment (recursivefunk) [inline src/scss/type.scss]: :tada:

**Comment by recursivefunk** [comment]
Code looks good :+1: for visual tickets, it might be helpful to attach an inline image so folks can see that the end result looks like. Maybe even record a gif and stick it in here. Just a thought

**Comment by oudeismetis** [comment]
Agree with Johnny's comment above. Pics would help. I also see only good things here :+1:

**Comment by camiriveron** [comment]
:)

screen shot 2016-02-16 at 4 54 24 pm

**Pull request merged**